upgrade/breaking-changes/list.md

```diff
@@ -10,4 +10,9 @@ position: 0
 
 This article lists the releases of the Telerik UI for .NET MAUI product that introduce breaking changes. You may want to go through it when [upgrading]({%slug upgrade-tutorial%}) to see whether you are affected.
```

Comment:
See the comments about this article in the review of https://github.com/telerik/maui-docs/pull/841.